### DISCUSSION TITLE
ActiveRecord::RecordInvalid: Validation failed: URL has already been taken

### DISCUSSION THREAD

**Discussion by Umiamz** [mastodon/mastodon#29101] [Q&A]
I see quite a lot of these in Sidekiq's dead jobs:

LinkCrawlWorker ActiveRecord::RecordInvalid: Validation failed: URL has already been taken

Anybody any idea what they mean and what causes them? This is a new server that I've just set up with 4.2.5 and I'm hoping that it's nothing I've done myself.

Thanks!

**Comment by danielcolquitt** [comment]
I am seeing the same thing with a clean install of vanilla mastodon v4.3.2.
